addons/netfox/rollback/rollback-synchronizer.gd

```diff
@@ -235,66 +271,105 @@ func _get_history(buffer: Dictionary, tick: int) -> Dictionary:
 	return buffer[before]
 
 @rpc("any_peer", "unreliable", "call_remote")
-func _submit_input(input: Dictionary, tick: int):
+func _submit_inputs(inputs: Dictionary, tick: int):
 	var sender_id: int = multiplayer.get_remote_sender_id()
 	var sanitized: Dictionary = {}
 
-	for property in input:
-		var pe: PropertyEntry = _property_cache.get_entry(property)
-		var value = input[property]
-		var input_owner_id: int = pe.node.get_multiplayer_authority()
-		
-		if input_owner_id != sender_id:
-			_logger.warning("Received input for node owned by %s from %s, sender has no authority!" \
-				% [input_owner_id, sender_id])
-			continue
-		
-		sanitized[property] = value
+	for picked_tick in inputs:
```

Comment:
I changed the logic of inputs because they were added `[property][tick]` instead of `[tick][property]`
This change is easier to read (see `_after_tick`) and more performant.

Again, if you want, I can remove this. Or move to a seperate PR. I only had it here because I planned to make input diffs as well, and that would require inputs to work with the same format as all other dictionaries, which is `[tick][properties]` instead of the current `[properties][tick]`